runtime/doc/term.txt

```diff
@@ -42,34 +43,34 @@ unlike most other environment variables.
 
   For this terminal           Set $TERM to                  |builtin-terms|
   -------------------------------------------------------------------------
+  anything libvte-based       vte, vte-256color                   Y
+   (e.g. GNOME Terminal)      (aliases: gnome, gnome-256color)
+  ConEmu                      conemu                              Y
+  cygwin                      cygwin                              Y
```

Comment:
@erw7 Is this for cygwin pty object, or when is it used? I assume it is not used for mintty, which I think typically sets `TERM=xterm-256color` .

Reply:
Use cygwin when terminal is `conhost`(It is when you use `Cygwin.bat` from `cmd.exe`.).

Reply:
Supplement. Using `TERM=cygwin` is for nvim compiled with Cygwin.